pkg/bigqueryanalysis/insertion.go

```diff
@@ -0,0 +1,58 @@
+package bigqueryanalysis
+
+import (
+	bigqueryv1 "github.com/openshift/sippy/pkg/apis/bigquery/v1"
+	sippyv1 "github.com/openshift/sippy/pkg/apis/sippyprocessing/v1"
+	"github.com/openshift/sippy/pkg/util"
+)
+
+// InsertBigQueryDataToJobs adds data from the bigquery tables to the JobResult data from TestGrid.
+func InsertBigQueryDataToJobs(bigqueryJobs []bigqueryv1.Job, sippyJobs []sippyv1.JobResult) []sippyv1.JobResult {
```

Comment:
Nit: mentally this feels like a merge, suggest MergeBigQueryDataToJobs or similar.